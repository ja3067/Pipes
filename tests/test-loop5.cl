for i in [1, 2, 3]:
    print(i)
    type(i) # dyn

type(i) # dyn
print(i) # 3

for i in "hello":
    print(i)
    type(i) # str

type(i) # dyn
print(i) # o

for i in range(5):
    print(i)
    type(i) # int

type(i) # dyn
print(i) # 4


