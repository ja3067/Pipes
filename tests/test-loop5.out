dyn
dyn
str
dyn
int
dyn
1
2
3
3
h
e
l
l
o
o
0
1
2
3
4
4
